.github/workflows/nessie.yaml

```diff
@@ -89,21 +82,21 @@ jobs:
       - name: Login to Amazon ECR
         id: login-ecr
         uses: aws-actions/amazon-ecr-login@v1
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
 
       - name: Build and push to Amazon ECR
+        # TODO(ariels): review & fork this?
+        uses: mr-smithers-excellent/build-push-action@v5
```

Comment:
Do we really want to give a custom action our creds? We just recently had a vulnerability issue with codecov.
I also dislike the custom shell script for checking, building and pushing.